### ISSUE TITLE
`company-flx` seems to be required.

### ISSUE THREAD

**Issue by joostkremers** [mguzmann/company-ipa#3]
This is really a great idea, I'm usually futzing with the X-Sampa input method...

Unfortunately, it seems to have a hard dependency on `company-flx`, because without it, it doesn't really work. I get  list of completion candidates that I can scroll through, but typing part of the name does not narrow the list of candidates.

I don't know enough about `company` to know if it can be made to work without `company-flx`, but if not, you should probably mention this in the README.

**Comment by mguzmann** [comment]
The alternative to using company-flx is having to type the exact name of the candidate, which I find super unpractical. Not sure what the solution would be. Do you have any ideas? BTW, this is now in melpa and the offical repo is over in gitlab. I'll keep this as a mirrow and update it soon.

**Comment by joostkremers** [comment]
> The alternative to using company-flx is having to type the exact name of the candidate, which I find super unpractical. Not sure what the solution would be. Do you have any ideas? 

I agree it's much more practical with flx, but an alternative without it would be if similar characters start with the same substring. Then you can use `TAB` to narrow the list of candidates and `M-n` / `M-p` to get to the desired symbol once the list is sufficiently small. Currently, that's not possible because the candidate strings have spaces in them.

I'm not sure it's worth the trouble, though. If you'd accept a PR, I might look into it some time next week.

> BTW, this is now in melpa

Yeah, I learnt about the package when I updated my package list. :-)

**Comment by mguzmann** [comment]
> I'm not sure it's worth the trouble, though. If you'd accept a PR, I might look into it some time next week.

Sure, but please use the gitlab repo: https://gitlab.com/mguzmann89/company-ipa/

**Comment by joostkremers** [comment]
> Sure, but please use the gitlab repo: https://gitlab.com/mguzmann89/company-ipa/

Will do.
